src/pages/contributors/yaroslav-afenkin.adoc

```diff
@@ -0,0 +1,96 @@
+= Yaroslav Afenkin
+:page-name: Yaroslav Afenkin
+:page-linkedin: yaroslavafenkin
+:page-twitter:
+:page-github: yaroslavafenkin
+:page-email:
+:page-image: avatar/yaroslav-afenkin.jpg
+:page-pronouns: He/Him/His
+:page-location: Odesa, Ukraine
+:page-firstcommit: 2018
```

Comment:
Is this first commit ever, or first commit to Jenkins? :o

Reply:
It would be the first commit to Jenkins. I tried to base it off the timeline we had during our conversation, as I forgot to ask about it directly.

Reply:
Okay great, I'll update that then. Thanks very much